16_wordle/main.go

```diff
@@ -48,6 +39,21 @@ func main() {
 	}
 }
 
+func NewList(path string) (List, error) {
+	rawWordList, err := os.ReadFile(path)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	wordList := bytes.Split([]byte(rawWordList), []byte("\n"))
+
+	validWords := List{
+		words: wordList,
+	}
+	return validWords, err
```

Comment:
here, the err will always be `nil`, so you can also do

```suggestion
	return validWords, nil
```

Reply:
Added